src/packages/@ncigdc/modern_components/SampleType/SampleType.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { compose } from 'recompose';
+
+import { withTheme } from '@ncigdc/theme';
+import withRouter from '@ncigdc/utils/withRouter';
+
+export default compose(withRouter, withTheme)(({ repository }) => {
+  return (
+    <td>
+      {repository.cases.hits.edges[0].node.samples.hits.edges[0].node.sample_type.toString()}
```

Comment:
shouldn't happen but can you use lodash get to provide a default value if the path is invalid? Something like `No sample type found`